managed/services/management/backup/artifacts_service.go

```diff
@@ -33,16 +36,18 @@ type ArtifactsService struct {
 	l          *logrus.Entry
 	db         *reform.DB
 	removalSVC removalService
+	storageSVC pitrStorage
```

Comment:
```suggestion
	removalSvc removalService
	storageSvc pitrStorage
```
Let's keep this consistent with other codebase